Sources/WinAppDriver/WinAppDriver.swift

```diff
@@ -34,12 +40,59 @@ public class WinAppDriver: WebDriver {
         executablePath: String = defaultExecutablePath,
         ip: String = defaultIp,
         port: Int = defaultPort,
-        waitTime: TimeInterval? = defaultStartWaitTime) throws -> WinAppDriver {
-
+        waitTime: TimeInterval? = defaultStartWaitTime,
+        outputFile: String? = nil) throws -> WinAppDriver {
         let processTree: Win32ProcessTree
+        var childStdinHandle: HANDLE? = nil
         do {
-            processTree = try Win32ProcessTree(path: executablePath, args: [ ip, String(port) ])
+            var launchOptions = ProcessLaunchOptions()
+            if let outputFile = outputFile {
+                // Open the output file for writing to the child stdout.
+                var securityAttributes = SECURITY_ATTRIBUTES()
+                securityAttributes.nLength = DWORD(MemoryLayout<SECURITY_ATTRIBUTES>.size)
+                securityAttributes.bInheritHandle = true
+                launchOptions.stdoutHandle = try outputFile.withCString(encodedAs: UTF16.self) {
+                    outputFile throws in
```

Comment:
Why throws if the body doesn't throw?

Reply:
Good point, I'm making a follow-up PR on that one.